LibTrixi.jl/src/api_c.jl

```diff
@@ -328,32 +373,60 @@ trixi_nvariables_cfptr() = @cfunction(trixi_nvariables, Cint, (Cint,))
 
 
 """
-    trixi_load_cell_averages(data::Ptr{Cdouble}, simstate_handle::Cint)::Cvoid
+    trixi_load_cell_averages(data::Ptr{Cdouble}, index::Cint, simstate_handle::Cint)::Cvoid
 
 Return cell averaged solution state.
 
-Cell averaged values for each cell and each primitive variable are stored in a contiguous
-array, where cell values for the first variable appear first and values for the other
-variables subsequently (structure-of-arrays layout).
+Cell averaged values for the primitive variable at position `index` for each cell are stored
+in the given array `data`.
 
 The given array has to be of correct size and memory has to be allocated beforehand.
```

Comment:
Maybe give a hint on how to determine the correct size that needs to be allocated?